### ISSUE TITLE
croplabel() , set pixel/voxel size and origin of the cropped image

### ISSUE THREAD

**Issue by romainGuiet** [ijpb/MorphoLibJ#63]
Dear @MorphoLibJ team,

Would it be possible to add the pixel/voxel size and the origin of the cropped image (maybe via Calibration ? )

https://github.com/ijpb/MorphoLibJ/blob/59b691fb8fa457e2fad8d5eb57c82c0fe6115e67/src/main/java/inra/ijpb/label/LabelImages.java#L329

This would ease a lot the use of this cropped image!

One workaround is to get the BoudingBox via "Analyze Regions 3D", but I'm a bit afraid by rounding issues ...

Cheers,

R

**Comment by dlegland** [comment]
hi Romain,
yes, sounds like a good idea! using Calibration looks the best way to do a priori.

**Comment by dlegland** [comment]
Hi,

I finally took some time to implement this! It was introduced in commit https://github.com/ijpb/MorphoLibJ/commit/de103336d363c5b889856fc53af9fd72f7a68f1b. The bounds are computed via a new "IntBounds2D/3D" class, that prevents from rounding errors.

By the way, I do not know why I closed the PR you opened? maybe I was too quick to cleanup branches... Anyway, I hope this (finally...) fits the need

Best,
David
